app/instance-initializers/bugsnag.js

```diff
@@ -49,12 +42,8 @@ export default {
 	},
 
 	_onError(error) {
-		if (!error) {
-			return;
-		}
-
 		if (!error || typeof error !== 'object' || !error.name || !error.message) {
```

Comment:
This should check the error is an error:

```suggestion
		if (error instanceof Error) {
```